test/Engine/FFITarget/FFITarget.csproj

```diff
@@ -22,9 +22,16 @@
         <Reference Include="Microsoft.CSharp" />
         <Reference Include="System.Data" />
         <Reference Include="System.Xml" />
+        <Reference Include="$(PkgMicrosoft_Office_Interop_Excel)\lib\net20\Microsoft.Office.Interop.Excel.dll">
```

Comment:
should this reference be wrapped in a condition for now?

Reply:
will this make it to the bin folder? I think we purposefully removed it at some point.

Reply:
totally forgot about test dlls. Will work on removing them

Reply:
done